cogs/make_applicant.py

```diff
@@ -49,7 +49,7 @@ async def _perform_make_applicant(self, ctx: TeXBotApplicationContext, applicant
             ephemeral=True,
         )
 
-        AUDIT_MESSAGE: Final[str] = f"{ctx.user} used TeX Bot Command \"Make User Applicant\""
+        AUDIT_MESSAGE: Final[str] = f'{ctx.user} used TeX Bot Command "Make User Applicant"'
```

Comment:
Prefer strings to use double quotes & escape interior double quotes rather than using single quotes